R/ResampleResult.R

```diff
@@ -80,4 +82,128 @@ fortify.ResampleResult = function(model, data, measure = NULL, ...) {
     variable.name = "measure_id", value.name = "performance")
 }
 
+plot_learner_prediction_resample_result = function(learner, object, measure, grid_points = 100L, expand_range = 0) {
+  UseMethod("plot_learner_prediction_resample_result")
+}
+
+plot_learner_prediction_resample_result.LearnerClassif = function(learner, object, measure, grid_points = 100L, expand_range = 0) {
+
+  task = object$task
+  features = task$feature_names
+  learners = object$learners
+
+  if (length(features) != 2) {
+    stop("plot_learner_prediction.TaskClassif only works for tasks with two features!")
+  }
+
+  grid_res = predict_grid(learners, task, grid_points = grid_points, expand_range = expand_range)
+
+  if (length(learners) > 1) {
```

Comment:
Do we need faceting? Would it be okay for the user to just call the plot function multiple times?

Reply:
I don't have any strong opinions on this. So we would allow to pass an argument e.g. `i` which allows to select the right learner? How would we call this argument? `fold` makes sense for CV but not for bootstrap etc. `resample.iter`, `split`, `split_i`, ...?